pages/1.11/security/ent/secrets/create-secrets/index.md

```diff
@@ -51,6 +51,8 @@ Secrets should include paths, unless you want to allow all services to access it
 
 1. Click **Create**.
 
+**Note:** It is not possible to create upload binary file using the GUI. Only CLI can be used to upload binary files.
```

Comment:
"to create upload binary file" I think there's an extra word here, maybe "upload a binary file"?

Reply:
"Only the CLI can be used ..."

Reply:
Changed:

> **Note:** It is not possible to create binary secret using the GUI. Only the CLI can be used to upload create binary secrets.

Reply:
> a binary secrets

Remove "a".